app/models/card.rb

```diff
@@ -13,61 +13,34 @@ class Card < ActiveRecord::Base
     where("review_date <= ?", DateTime.current).order("RANDOM()")
   }
 
-  def check_translation(user_input)
+  def check_translation(user_input, time_to_answer)
```

Comment:
Unused method argument - `time_to_answer`. If it's necessary, use `_` or `_time_to_answer` as an argument name to indicate that it won't be used.